### PR TITLE
Fix visual borking in the share dialog

### PR DESCRIPTION
**Pull request by claucambra**
Now:

image

Before:

image

### CODE REVIEW

**Comment by allexzander** [comment]
@claucambra Should not the magnifying glass inside the text field also be white in dark mode?

File: src/gui/sharelinkwidget.cpp [hunk 1]
```diff
@@ -593,9 +593,13 @@ void ShareLinkWidget::slotStyleChanged()
 
 void ShareLinkWidget::customizeStyle()
 {
-    _unshareLinkAction->setIcon(Theme::createColorAwareIcon(":/client/theme/delete.svg"));
+    if(!_unshareLinkAction.isNull()) {
```
Comment (mgallien) [inline src/gui/sharelinkwidget.cpp:596]: this test looks unusual in our code base, maybe you could simply use `if(!_unshareLinkAction) {` ?

**Comment by claucambra** [comment]
> @claucambra Should not the magnifying glass inside the text field also be white in dark mode?

Fixed now

File: src/gui/sharelinkwidget.cpp (hunk 1 above)
Comment (claucambra) [reply src/gui/sharelinkwidget.cpp]: Thanks, using our usual style now

**Review by mgallien — APPROVED** [review]

**Pull request merged**